src/commands/installCosmosDBExtension.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { commands } from 'vscode';
+import { AzureTreeItem, IActionContext } from 'vscode-azureextensionui';
+import { CosmosDBTreeItem } from '../explorer/CosmosDBTreeItem';
+import { delay } from '../utils/delay';
+import { openUrl } from "../utils/openUrl";
+
+export async function installCosmosDBExtension(context: IActionContext, treeItem: AzureTreeItem): Promise<void> {
+    const commandToRun = 'extension.open';
+    const listOfCommands = await commands.getCommands();
+    if (listOfCommands.find((x: string) => x === commandToRun)) {
```

Comment:
You can just use the command without doing all this extra validation. Our minimum engine is 1.31.0 in package.json and the command was added in September 2018 (https://github.com/microsoft/vscode/issues/58642) which looks like it was 1.28.0